world/templates/template_commands.py

```diff
@@ -86,17 +90,38 @@ def func(self):
             if not args:
                 super(CmdTemplateForm, self).func()
             else:
-                template = self.find_template(args)
-                if template:
-                    self.caller.msg(self.display(template))
+                try:
+                    template = Template.objects.filter(id=args)[:1].get()
```

Comment:
```suggestion
                    template = Template.objects.get(id=args)
```
is a little more readable to me, though should be the same thing